src/utils/PatchError.js

```diff
@@ -52,8 +52,8 @@ export default class PatchError extends Error {
     for (let line = displayStartLine; line <= displayEndLine; line++) {
       let startOfLine = lineMap.indexForLocation({ line, column: 0 });
       let endOfLine = lineMap.indexForLocation({ line: line + 1, column: 0 });
-      if (isNaN(endOfLine)) {
-        if (isNaN(startOfLine)) {
+      if (endOfLine == null) {
```

Comment:
Ok, I just updated it. FYI I saw that the flow type for `indexForLocation` is `?number`, which means "undefined or null or a number" (I think), so I've found that a `== null` check (i.e. checking for `null` and `undefined` at the same time) tends to make flow happier. Not sure if that really matters in this case, though.